contentctl/objects/unit_test.py

```diff
@@ -17,19 +16,21 @@ class UnitTest(BaseTest):
     # contentType: SecurityContentType = SecurityContentType.unit_tests
 
     # The test type (unit)
-    test_type: TestType = Field(TestType.UNIT)
+    test_type: TestType = Field(default=TestType.UNIT)
 
+    # TODO (cmcginley): looks like pass_condition has gone the way of the dodo; can I remove?
     # The condition to check if the search was successful
-    pass_condition: Union[str, None] = None
+    pass_condition: str | None = None
 
+    # TODO (cmcginley): looks like baselines has gone the way of the dodo; can I remove?
     # Baselines to be run before a unit test
     baselines: list[UnitTestBaseline] = []
```

Comment:
Looks like baselines has gone the way of the dodo; can I remove?

Reply:
Do we have an issue or ticket which scopes this work?

Reply:
There is now an ISSUE for this: https://github.com/splunk/contentctl/issues/269